protocol-units/da/m1/util/src/ir_blob.rs

```diff
@@ -95,38 +95,38 @@ impl InnerSignedBlobV1 {
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub enum InnerBlob {
+pub enum IntermediateBlobRepresentation {
```

Comment:
Now you went overboard, I think `IntermediateBlob` would be fine, but OK :)

Reply:
Do you want me to make this change or just stick with the above?

Reply:
Whatever feels right.